packages/codemirror/src/theme.ts

```diff
@@ -54,6 +54,10 @@ export const jupyterEditorTheme = EditorView.theme({
     backgroundColor: 'var(--jp-layout-color2)'
   },
 
+  '.cm-gutter, .cm-lineNumbers': {
```

Comment:
Question: do we need `.cm-gutter` here? It looks that both classes are on the same element:

![Screenshot from 2023-04-01 17-01-09](https://user-images.githubusercontent.com/5832902/229301489-f9dff2b1-609f-4a45-839a-411db994be09.png)

If that is not needed, then fewer CSS rules is better.

```suggestion
  '.cm-lineNumbers': {
```

Reply:
`  '.cm-gutter, .cm-lineNumbers'` selector was already in place so I considered that we might be planning to show line numbers in some other place except gutter. If it's not the case, just ` '.cm-lineNumbers':` should work